src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -483,7 +483,7 @@ private function validateAndDenormalize(string $currentClass, string $attribute,
                     return (float) $data;
                 }
 
-                if (('is_'.$builtinType)($data)) {
+                if ('false' === $builtinType || ('is_'.$builtinType)($data)) {
```

Comment:
I've inlined the value of the constant: the minimum version bump is not worth it IMHO.

Reply:
Alright, will do it as well for https://github.com/php/php-src/pull/8326 when it's accepted. Thanks :+1: